Restored graphviz config throws error

h3. How to reproduce

 * (with current main-branch, not a release) Analyse a project using any of the graphviz formatters _without_ formatters config in the depfile, e.g. `docs/examples/ControllerServiceRepository1.depfile.yaml`

*Expected behavior*

Defaults are used and image will be rendered:

```
groups = array()
hidden_layers = array()
point_to_groups=false
```

*Actual behavior*

Exception is thrown:

```
PHP Warning:  Undefined array key "hidden_layers" in phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php on line 18

Warning: Undefined array key "hidden_layers" in phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php on line 18
PHP Warning:  Undefined array key "groups" in phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php on line 18

Warning: Undefined array key "groups" in phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php on line 18
PHP Warning:  Undefined array key "point_to_groups" in phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php on line 18

Warning: Undefined array key "point_to_groups" in phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php on line 18


 [ERROR]

         Output formatter graphviz-image threw an Exception:

         Message: Qossmic\Deptrac\Supportive\OutputFormatter\Configuration\ConfigurationGraphViz::__construct():
         Argument #1 ($hiddenLayers) must be of type array, null given, called in
         phar:///Users/dbr/Projects/qossmic/deptrac-shim/deptrac/src/Supportive/OutputFormatter/Configuration/Configurat
         ionGraphViz.php on line 18
```

On the `main` as well as `1.0.2` branch, running on `deprac` itself:
```console
$ patrick@thinkpad ~/PhpstormProjects/deptrac (main)$ php deptrac.php analyse -f graphviz-dot
 235/235 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


                                                                                                                        
 [ERROR]                                                                                                                
                                                                                                                        
         Output formatter graphviz-dot threw an Exception:                                                              
                                                                                                                        
         Message: No '--output' defined for GraphViz formatter                                                          
                                                                                                                        
                                                                                                                        
                                                                                                                        


$ patrick@thinkpad ~/PhpstormProjects/deptrac (main)$ php deptrac.php analyse -f graphviz-dot --output test.dot
 235/235 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

Script dumped to /home/patrick/PhpstormProjects/deptrac/test.dot
$ patrick@thinkpad ~/PhpstormProjects/deptrac (main)$ 
```

--- a/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php
+++ b/src/Supportive/OutputFormatter/Configuration/ConfigurationGraphViz.php
@@ -7,11 +7,11 @@ namespace Qossmic\Deptrac\Supportive\OutputFormatter\Configuration;
 final class ConfigurationGraphViz
 {
     /**
-     * @param array{hidden_layers: string[], groups: array<string, string[]>, point_to_groups: bool} $arr
+     * @param array{hidden_layers?: string[], groups?: array<string, string[]>, point_to_groups?: bool} $arr
      */
     public static function fromArray(array $arr): self
     {
-        return new self($arr['hidden_layers'], $arr['groups'], $arr['point_to_groups']);
+        return new self($arr['hidden_layers'] ?? [], $arr['groups'] ?? [], $arr['point_to_groups'] ?? false);
     }
 
     /**

--- a/src/Supportive/OutputFormatter/GraphVizOutputFormatter.php
+++ b/src/Supportive/OutputFormatter/GraphVizOutputFormatter.php
@@ -26,13 +26,13 @@ use function tempnam;
 abstract class GraphVizOutputFormatter implements OutputFormatterInterface
 {
     /**
-     * @var array{hidden_layers: string[], groups: array<string, string[]>, point_to_groups: bool}
+     * @var array{hidden_layers?: string[], groups?: array<string, string[]>, point_to_groups?: bool}
      */
     private readonly array $config;
 
     public function __construct(FormatterConfiguration $config)
     {
-        /** @var array{hidden_layers: string[], groups: array<string, string[]>, point_to_groups: bool}  $extractedConfig */
+        /** @var array{hidden_layers?: string[], groups?: array<string, string[]>, point_to_groups?: bool}  $extractedConfig */
         $extractedConfig = $config->getConfigFor('graphviz');
         $this->config = $extractedConfig;
     }

